kde-plasma/kinfocenter/kinfocenter-6.0.49.9999.ebuild

```diff
@@ -34,7 +34,6 @@ DEPEND="
 	usb? ( virtual/libusb:1 )
 "
 RDEPEND="${DEPEND}
-	dev-qt/qdbus:*
```

Comment:
`>=dev-qt/qttools-${QTMIN}:6[qdbus]` is not needed? Did you check that?

Reply:
The main binary links to these:
 
0x0000000000000001 (NEEDED)             Shared library: [libsystemsettingsview.so.3]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6ItemModels.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6XmlGui.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6DBusAddons.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6KCMUtilsCore.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6ItemViews.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6KIOGui.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6WindowSystem.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6KIOCore.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6Crash.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6IconThemes.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6ConfigWidgets.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6I18n.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6WidgetsAddons.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6Service.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6ConfigCore.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libKF6CoreAddons.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libQt6QuickWidgets.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libQt6Widgets.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libQt6Quick.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libQt6Qml.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libQt6Gui.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libQt6Core.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libstdc++.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
 0x000000006ffffffb (FLAGS_1)            Flags: PIE

The plugins link to these:

/usr/lib64/qt6/plugins/plasma/kcms/kinfocenter link to these:

[libc.so.6]
[libKF6CoreAddons.so.6]
[libKF6I18n.so.6]
[libKF6KCMUtilsCore.so.6]
[libKF6KCMUtilsQuick.so.6]
[libKF6KCMUtils.so.6]
[libKF6KIOCore.so.6]
[libKF6KIOGui.so.6]
[libKF6KIOWidgets.so.6]
[libKF6Service.so.6]
[libKF6Solid.so.6]
[libKInfoCenterInternal.so]
[libQt6Core.so.6]
[libQt6DBus.so.6]
[libQt6Qml.so.6]
[libQt6Widgets.so.6]
[libstdc++.so.6]
[libusb-1.0.so.0]

libQt6DBus is in qtbase so it should all be fine

Reply:
But this is about the qdbus binary used at runtime, hence being a runtime dependency only.

Reply:
https://invent.kde.org/plasma/kinfocenter/-/blob/Plasma/6.0/Modules/kwinsupportinfo/kcm_kwinsupportinfo.json.in?ref_type=heads#L88